internal/signaling/types.go

```diff
@@ -67,6 +67,13 @@ type JoinedPacket struct {
 	LobbyInfo stores.Lobby `json:"lobbyInfo"`
 }
 
+type LeaderPacket struct {
+	Type string `json:"type"`
+
```

Comment:
Idea: Should this packet also contain the lobby it's about?

Reply:
Since you can currently only be in 1 lobby that wouldn't make sense.

Reply:
Fair, I figured it would make it forward compatible. But I guess we can always add it